app/src/main/java/com/futurice/freesound/feature/home/HomeFragmentViewModel.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.feature.home
+
+import com.futurice.freesound.network.api.model.User
+import com.futurice.freesound.viewmodel.SimpleViewModel
+import io.reactivex.Single
+
+internal class HomeFragmentViewModel(userDataModel: UserDataModel) : SimpleViewModel() {
+
+    private val homeUser: Single<User> = userDataModel.homeUser.cache()
+
+    val image: Single<String>
+        get() = homeUser.map { it.avatar().large() }
```

Comment:
Can you explain the benefits of using property notation here, rather than just having methods named `image`, `username` etc.